src/masternodes/rpc_vault.cpp

```diff
@@ -0,0 +1,707 @@
+#include <masternodes/mn_rpc.h>
+
+extern UniValue AmountsToJSON(TAmounts const & diffs);
+extern std::string tokenAmountString(CTokenAmount const& amount);
+
+namespace {
+    UniValue BatchToJSON(const CVaultId& vaultId, uint32_t batchCount) {
+        UniValue batchArray{UniValue::VARR};
+        for (uint32_t i = 0; i < batchCount; i++) {
+            UniValue batchObj{UniValue::VOBJ};
+            auto batch = pcustomcsview->GetAuctionBatch(vaultId, i);
+            batchObj.pushKV("index", int(i));
+            batchObj.pushKV("collaterals", AmountsToJSON(batch->collaterals.balances));
+            batchObj.pushKV("loan", tokenAmountString(batch->loanAmount));
+            batchArray.push_back(batchObj);
+        }
+        return batchArray;
+    }
+
+    UniValue VaultToJSON(const CVaultId& vaultId, const CVaultData& vault) {
+        UniValue result{UniValue::VOBJ};
+        result.pushKV("loanSchemeId", vault.schemeId);
```

Comment:
```suggestion
        result.pushKV("vaultId", vaultId.GetHex());
        result.pushKV("loanSchemeId", vault.schemeId);
```
for consistency with other get* RPC